### PR TITLE
Fix TestClient unable to get replicasets

### PR DESCRIPTION
**Pull request by linkous8**
Resolves https://github.com/vapor-ware/kubetest/issues/213

### CODE REVIEW

File: kubetest/objects/replicaset.py [hunk 1]
```diff
@@ -39,8 +39,6 @@ def __init__(self, *args, **kwargs) -> None:
         super(ReplicaSet, self).__init__(*args, **kwargs)
         self._add_kubetest_labels()
 
-        client.AppsV1Api.read_namespaced_replica_set()
```
Comment (rstarmer) [inline kubetest/objects/replicaset.py:42]: @linkous8 I presume this fails on init?

is there another test that should address this (perhaps it's not even important?)

File: kubetest/client.py [hunk 2]
```diff
@@ -971,7 +971,7 @@ def get_replicasets(
         replicasets = {}
         for obj in results.items:
             rs = objects.ReplicaSet(obj)
-            replicasets[replicasets.name] = rs
```
Comment (rstarmer) [inline kubetest/client.py:974]: without this change, the returned value would be {}

**Review by rstarmer — APPROVED** [review]
@blakewatters @pnickolov @mjdeclerck 

This looks good to me, one question on the replicasets.py, but shouldn't block anything. 

I recommend merging.

File: kubetest/objects/replicaset.py (hunk 1 above)
Comment (linkous8) [reply kubetest/objects/replicaset.py:42]: Correct, it raises an error for missing parameters. The updated implementation brings it into line with the other controller models implemented by this library. Eg. :
https://github.com/opsani/kubetest/blob/f77c5c8056d9ad51a699312874b350cff3f77529/kubetest/objects/deployment.py#L39-L42
https://github.com/opsani/kubetest/blob/f77c5c8056d9ad51a699312874b350cff3f77529/kubetest/objects/statefulset.py#L39-L42

As for testing, this library only implements unit testing for model instantiation as well as loading from manifests. I was unable to locate any tests that interact with live objects in a cluster

File: kubetest/client.py (hunk 2 above)
Comment (linkous8) [reply kubetest/client.py:974]: `replicasets` is a dictionary which has no name property. I'm pretty sure this was just a typo

**Pull request merged**